src/clients/slaos.py

```diff
@@ -0,0 +1,38 @@
+from datetime import datetime
+import httpx
+
+from src.config.models.offset import SLIIndexedDocument
+from src.config.models.output import RatedOutputConfig
```

Comment:
I've used another area's config object here. There is 100% overlap in config. I should probably restructure the config yaml instead.